frontend/src/components/3.CreateGroup/CreateGroup.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {useState} from 'react';
+import Help from '../Common/Help';
+import '../Common/Help.css';
+// import styled from 'styled-components';
+
+/**
+ *
+ * @return {*}
+ */
+function makeid() {
+  let result = '';
+  const characters =
+    'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  for (let i = 0; i < 5; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+/**
+ *
+ * @return {*}
+ */
+function CreateGroup() {
+  const [ButtonPopup, setButtonPopup] = useState(false);
+
+  return (
+    <>
+      <h1 className="Title">yumble</h1>
+      <div className="MakeCentre">
+        <div className="GroupCode_Box">
+          Your group code is...
+          <div style={{padding: '1vw'}}>{makeid()}</div>
```

Comment:
Similar to above - I think that inline styling should be pulled out into a stylesheet ideally so that we don't have unpredictable css hierarchies